service/test/controllers/skills/validator/UserSkillRequestValidatorTest.java

```diff
@@ -0,0 +1,76 @@
+package controllers.skills.validator;
+
+import static org.junit.Assert.assertEquals;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.junit.Assert;
+import org.junit.Test;
+import org.powermock.core.classloader.annotations.PowerMockIgnore;
+import org.sunbird.common.exception.ProjectCommonException;
+import org.sunbird.common.models.util.JsonKey;
+import org.sunbird.common.request.Request;
+
+/** Created by rajatgupta on 21/08/18. */
+@PowerMockIgnore({"javax.management.*", "javax.net.ssl.*", "javax.security.*"})
+public class UserSkillRequestValidatorTest {
+
+  @Test
+  public void validateSkillUpdateSuccess() {
+    Request request = new Request();
+    boolean response = false;
+    Map<String, Object> requestObj = new HashMap<>();
```

Comment:
Create a private method which constructs the request based on input to avoid code duplication.

Map<String, Object> createRequest(String userId, List<String> skills, String requestedByUserId)